parser/parser.y

```diff
@@ -864,13 +871,16 @@ ColumnOption:
 	{
 		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionAutoIncrement}
 	}
-|	"PRIMARY" "KEY"
+|	PrimaryOpt "KEY" 
 	{
+		// KEY is normally a synonym for INDEX. The key attribute PRIMARY KEY 
+		// can also be specified as just KEY when given in a column definition. 
+		// See: http://dev.mysql.com/doc/refman/5.7/en/create-table.html
 		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionPrimaryKey}
 	}
-|	"UNIQUE"
+|	"UNIQUE" %prec lowerThanKey
```

Comment:
Yes "unique key" can be treat as "unique key" or "unique"+"key", so we must specify its precedence.